ChangeLog.md

```diff
@@ -20,6 +20,9 @@ See docs/process.md for more on how version tagging works.
 
 3.1.54 (in development)
 -----------------------
+- The `DEMANGLE_SUPPORT` setting and the associated `demangle` function are
+  now deprecated since Wasm stack traces no longer contain mangled symbol
+  names under any circumstances. (#21346)
```

Comment:
```suggestion
- The `DEMANGLE_SUPPORT` setting and the associated `demangle` function are
  now deprecated since Wasm stack traces are always demangled now anyhow. (#21346)
```